packages/vats/src/core/startWalletFactory.js

```diff
@@ -191,7 +192,11 @@ export const startWalletFactory = async (
     feeIssuerP,
   ]);
 
-  const oldAddresses = harden(chainStorageEntries.map(entry => entry[0]));
+  // Carry forward wallets with an address already in chain storage.
+  const dataReviver = makeHistoryReviver(chainStorageEntries);
+  const walletStoragePath = await E(walletStorageNode).getPath();
```

Comment:
If someone chose to deploy the new contract with a different storage path, that won't change where the old data is. Hard-coding the `published.wallet` path seems appropriate.

Reply:
Even though nothing else in this file has awareness of the "published" prefix? I was thinking that if the storage path ever changed, we'd introduce a vat parameter to communicate the old path.

Reply:
In practice I suppose it doesn't matter.